src/Utilities/ArrayExtension.cs

```diff
@@ -1,9 +1,16 @@
 ﻿using System;
+using System.Collections.Generic;
+using ePiggyWeb.DataManagement.Entries;
 
 namespace ePiggyWeb.Utilities
 {
     public static class ArrayExtension
     {
+        public static IEntryList ToIEntryList(this IEnumerable<IEntry> entryEnumerable)
+        {
+            return new EntryList(entryEnumerable);
```

Comment:
How does this overcome problem of not being able to cast it simply?

Reply:
Well as I understand, casting collections of objects is different than casting objects, for example you usually see methods like "ToArray", "ToList", etc., so I instead of trying to figure out a convoluted cast, I just followed this simple pattern and it seems like a much neater and easier solution, at least to me :D